tests/test_model.py

```diff
@@ -253,3 +253,16 @@ def test_model_add_param(self):
         # Assert
         self.assertHasAdditions(source_file, self.result, ['parameter Real myParam=1.25 "a comment"'])
         self.assertNoDeletions(source_file, self.result)
+
+    def test_model_add_param_type_string(self):
+        # Setup
+        source_file = os.path.join(self.data_dir, 'DCMotor.mo')
+        model = Model(source_file)
+
+        # Act
+        model.add_parameter('String', 'myParam', string_comment='a comment', assigned_value='"supercalifragilisticexpialidocious"')
```

Comment:
yes I checked the spelling of supercalifragilisticexpialidocious. https://youtu.be/qkHV6THFoDs?t=51

Reply:
I googled it 😆